src/components/AttachmentModal.js

```diff
@@ -18,6 +18,7 @@ import fileDownload from '../libs/fileDownload';
 import withLocalize, {withLocalizePropTypes} from './withLocalize';
 import ConfirmModal from './ConfirmModal';
 import TextWithEllipsis from './TextWithEllipsis';
+import getLocalFileUri from '../libs/getLocalFileUri';
```

Comment:
```suggestion
import getLocalFileUri from '../libs/getLocalFileUri';
```

Rename this to getDecodedURIPath and the rename the file to same